.changeset/flatten-sidebar-islands.md

```
@@ -0,0 +1,6 @@
+---
+'@ciderpress/ui': patch
+---
+
+Flatten isolated sidebar sections instead of rendering their root as a dropdown. A mapped island
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)